examples/webgl_gpu_picking_point.html

```diff
@@ -0,0 +1,298 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>three.js webgl - gpu - picking - point</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+		body {
+			color: #cccccc;
+			font-family: Monospace;
+			font-size: 13px;
+			text-align: center;
+
+			background-color: #050505;
+			margin: 0px;
+			overflow: hidden;
+		}
+
+		#info {
+			position: absolute;
+			top: 0px;
+			width: 100%;
+			padding: 5px;
+		}
+
+		a {
+			color: #0080ff;
+		}
+
+	</style>
+</head>
+<body>
+
+<div id="container"></div>
+<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - gpu - picking - point
+</div>
+
+<script type="module">
+
+	import * as THREE from '../build/three.module.js';
+	// import {WEBGL} from './js/WebGL.js';
+	import Stats from './jsm/libs/stats.module.js';
+	import {GUI} from './jsm/libs/dat.gui.module.js';
+	import {OrbitControls} from './jsm/controls/OrbitControls.js';
+
+	// if (WEBGL.isWebGLAvailable() === false) {
+	//
+	// 	document.body.appendChild(WEBGL.getWebGLErrorMessage());
+	//
+	// }
+
+	var renderCamera, scene, renderer, stats, pickingScene, gui;
+
+	var mesh;
+	var mesh1;
+
+	var controls;
+	let raycaster;
+
+	var pickingCamera
+
+	var GPUPickingPoint = new THREE.Vector3();
+	var CPUPickingPoint = new THREE.Vector3();
+
+	init();
+	animate();
+
+	function init() {
+
+		//
+		raycaster = new THREE.Raycaster();
+		renderCamera = new THREE.PerspectiveCamera(60, window.innerWidth / window.innerHeight, 1, 10000);
+		pickingCamera = new THREE.PerspectiveCamera(60, window.innerWidth / window.innerHeight, 1, 10000);
+		renderCamera.up.set(0, 0, 1);
+		pickingCamera.up.set(0, 0, 1);
+		renderCamera.position.x = 64;
+
+		scene = new THREE.Scene();
+		pickingScene = new THREE.Scene();
+		// scene.background = new THREE.Color(0x050505);
+
+		//
+		let axesHelper = new THREE.AxesHelper(100);
+		scene.add(axesHelper);
+
+		// var ambientLight = new THREE.AmbientLight(0x222222);
+		// scene.add(ambientLight);
+		//
+		// var light1 = new THREE.DirectionalLight(0xffffff, 0.5);
+		// light1.position.set(1, 1, 1);
+		// scene.add(light1);
+		//
+		// var light2 = new THREE.DirectionalLight(0xffffff, 1);
+		// light2.position.set(0, -1, 0);
+		// scene.add(light2);
+
+		//
+
+		// let geometry = new THREE.SphereBufferGeometry(100, 20, 20);
+		let geometry = new THREE.BoxBufferGeometry(100, 100, 100);
+
+		var depthMaterial = new THREE.MeshDepthMaterial();
+		depthMaterial.depthPacking = THREE.RGBADepthPacking;
+
+		mesh = new THREE.Mesh(geometry, depthMaterial);
```

Comment:
Why using `MeshDepthMaterial`? It's intended for certain use cases but not as a typical material for rendering meshes. Use e.g.  `MeshBasicMaterial`, `MeshLamberMaterial` or `MeshPhongMaterial` instead.

Reply:
I can directly see the depth map effect used in the off-screen frame.

Reply:
Um, I don't think this setup is good. It looks like a broken example.